src/prepack-cli.js

```diff
@@ -43,7 +43,9 @@ function run(
     --srcmapIn          The input sourcemap filename. If present, Prepack will output a sourcemap that maps from
                         the original file (pre-input sourcemap) to Prepack's output
     --srcmapOut         The output sourcemap filename.
-    --maxStackDepth     Specify the maximum call stack depth
+    --maxStackDepth     Specify the maximum call stack depth.
+    --timeout           The amount of time until Prepack should time out.
+    --addtlFunctions    Additional functions that should be prepacked (comma separated).
```

Comment:
No funky abbreviations please, especially if they don't even match the actual parameter name!

Reply:
Oops! Fixed